adapters/wagmi-connector/src/bloctoConnector.ts

```diff
@@ -0,0 +1,131 @@
+import { Connector, Chain, ConnectorData, SwitchChainError } from 'wagmi'
+import type { 
+  EthereumProviderConfig as BloctoOptions,
+  EthereumProviderInterface as BloctoProvider
+} from '@blocto/sdk';
+import BloctoSDK from '@blocto/sdk'; 
+import { providers } from 'ethers';
+import { ConnectorNotFoundError } from 'wagmi'
+import { hexValue } from 'ethers/lib/utils.js';
+
+type BloctoWalletSigner = providers.JsonRpcSigner;
+
+class BloctoConnector extends Connector<
+  BloctoProvider,
+  BloctoOptions,
+  BloctoWalletSigner
+> {
+
+  readonly id = 'bloctoWallet'
+  readonly name = 'Blocto Wallet'
+  readonly ready = true
+
+  #provider?: BloctoProvider;
+
+  constructor(config: { chains?: Chain[]; options: BloctoOptions }) {
+    super(config)
+  }
+
+  getProvider() {
+    if (!this.#provider) {
+      this.#provider = new BloctoSDK({ ethereum: this.options })?.ethereum;
+    }
+
+    if (!this.#provider) {
+      throw new ConnectorNotFoundError();
+    }
+
+    return Promise.resolve(this.#provider);
+  }
+  
+  async connect(config?: { chainId?: number | undefined; } | undefined): Promise<Required<ConnectorData<BloctoProvider>>> {
+    const provider = await this.getProvider();
+    await provider?.enable();
+    const account = await this.getAccount();
+    const id = await this.getChainId();
+    const unsupported = this.isChainUnsupported(id);
+
+    return {
+      account,
+      chain: { id, unsupported },
+      provider
+    }
+  }
+
+  async disconnect(): Promise<void> {
+    const provider = await this.getProvider();
+    await provider?.request({ method: "wallet_disconnect" });
+  }
+  
+  async getAccount(): Promise<`0x${string}`> {
+    const provider = await this.getProvider();
+
+    return provider
+      .request({ method: 'eth_requestAccounts' })
+      .then((accounts: string[]): `0x${string}` => accounts[0] as `0x{string}`)
```

Comment:
Should be as `0x${string}`

Reply:
[fix(BloctoConnector): correct getAccount() return type](https://github.com/portto/blocto-sdk/pull/71/commits/1b5280823dc6807e25e963ac918a24ea87d28c59) resolved.